ui/app/components/auth-config-form/config.js

```diff
@@ -11,23 +11,21 @@ const AuthConfigBase = Component.extend({
   router: service(),
   wizard: service(),
   saveModel: task(function*() {
-    yield this.model
-      .save()
-      .then(() => {
-        if (this.wizard.currentMachine === 'authentication' && this.wizard.featureState === 'config') {
-          this.wizard.transitionFeatureMachine(this.wizard.featureState, 'CONTINUE');
-        }
-        this.router.transitionTo('vault.cluster.access.methods').followRedirects();
-        this.flashMessages.success('The configuration was saved successfully.');
-      })
-      .catch(err => {
-        // AdapterErrors are handled by the error-message component
-        // in the form
-        if (err instanceof DS.AdapterError === false) {
-          throw err;
-        }
-        return;
-      });
+    try {
+      yield this.model.save();
+      if (this.wizard.currentMachine === 'authentication' && this.wizard.featureState === 'config') {
```

Comment:
I think it's likely not a huge deal, but this block can go after the catch - that way the try isn't too broad and ignoring errors we don't intend to (the catch will only throw ember-data related errors).